sklearn/semi_supervised/label_propagation.py

```diff
@@ -350,6 +373,14 @@ class LabelPropagation(BaseLabelPropagation):
     LabelSpreading : Alternate label propagation strategy more robust to noise
     """
 
+    variant = 'propagation'
```

Comment:
Please make this a private attribute `_variant`. Otherwise it could be consider public API.

@jnothman I think we can merge once this is made private. Ok on your side?